app/containers/Applications/ApplicationListContainer.tsx

```diff
@@ -10,13 +10,14 @@ import {SortOnlyOption} from 'components/Search/SortOnlyOption';
 import {EnumSearchOption} from 'components/Search/EnumSearchOption';
 import {CiipApplicationRevisionStatus} from 'createApplicationRevisionStatusMutation.graphql';
 import {ApplicationListContainer_query} from 'ApplicationListContainer_query.graphql';
+import FilterableTablePagination from 'components/FilterableComponents/FilterableTablePagination';
 
 interface Props {
   query: ApplicationListContainer_query;
 }
 
 export const ApplicationList: React.FunctionComponent<Props> = (props) => {
-  const {edges} = props.query.allApplications;
+  const {edges, totalCount} = props?.query?.allApplications;
```

Comment:
```suggestion
  const {edges, totalCount} = props.query.allApplications;
```
neither props nor query can be null